lib/chef-dk/command/verify.rb

```diff
@@ -281,6 +281,19 @@ def format_gem_failure(name, versions)
         end
       end
 
+      add_component "generated-cookbooks-pass-chefspec" do |c|
+
+        c.base_dir = "chef-dk"
+        c.smoke_test do
+          tmpdir do |cwd|
+            sh("chef generate cookbook example", cwd: cwd)
+            cb_cwd = File.join(cwd, "example")
+            sh("cat spec/spec_helper.rb", cwd: cb_cwd)
```

Comment:
good catch, it was for debugging as originally I forgot to make it cwd into the cookbook instead of just the tempdir.